### ISSUE TITLE
NeXML generated by OntoTrace uses HTTP URIs as document-local identifiers

### ISSUE THREAD

**Issue by hlapp** [phenoscape/phenoscape-kb-services#15]
The NeXML generated by OntoTrace includes elements such as the following:
```xml
<otu id="http://purl.obolibrary.org/obo/VTO_0036225" label="Ictalurus punctatus" about="#http://purl.obolibrary.org/obo/VTO_0036225">
```

Note the hash (`#`) preceding the HTTP URI in the `about` attribute. Is this intentional, and why is it needed?

**Comment by balhoff** [comment]
The OTU has the URI as an ID as a convenient document-unique ID which also references to the taxon. The about is there to say what the `<meta/>` inside refer to, which is the element with that ID in the current document. Is it causing a problem somehow?

**Comment by balhoff** [comment]
I downloaded Oxygen (which I have alway found to be the most reliable XML tool) to check these. It does look like the URIs are invalid ID values, specifically due to the colons and slashes. These IDs are only used in OntoTrace and not in Phenex. This does need to be fixed.

**Comment by rvosa** [comment]
The hash is intentional and needed, conform the first bullet item example in this document: http://www.w3.org/2006/07/SWD/RDFa/syntax/#id103742

Basically, it is needed to reconcile the identification of the subject in the RDFa triple with the equivalent XML element.

**Comment by balhoff** [comment]
Fixed by 201e8da8c214b80e5f906f5cdb01d5dac1552c09.
